src/Analyzer.Core.BuiltInRuleTests/Tests/AppServiceApiApp_RestrictCORSAccess.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "Template": "AppServices",
+        "ReportedFailures": [
+            {
```

Comment:
Same discussion as in Teams about rules with almost repeated code, I assume we will handle this situation later?

Reply:
We can discuss options here or open an issue and decide later. Thoughts?

Reply:
I like the second option :)

Reply:
Make a new issue https://github.com/Azure/template-analyzer/issues/172